### PR TITLE
Feature/4962 overview page

### PR DESCRIPTION
**Pull request by jesuspg**
@pratid

### CODE REVIEW

File: dashboard/lib/public/template.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Sanity Check Result Report- Fiware Region Status</title>
```
Comment (flopezag) [inline dashboard/lib/public/template.html]: Report - Fiware

**Comment by flopezag** [comment]
LGTM

File: dashboard/lib/routes/cbroker.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+'use strict';
+
+var http = require('http'),
+    cbroker = {};
+
+
+var FIELDNAME = 'sanity_status'; // field name for value about regions status
```
Comment (pratid) [inline dashboard/lib/routes/cbroker.js]: I'd rename this to something like `SANITY_STATUS_ATTRIBUTE`

File: dashboard/lib/routes/cbroker.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+
+/**
+ * @function parseRegions
+ *
+ * @param {Object} entities
+ * @return {Array}
+ */
+cbroker.parseRegions = function (entities) {
+
```
Comment (pratid) [inline dashboard/lib/routes/cbroker.js:34]: This is more a private function than a method. Why not define it as `function parseRegions(entities)`?

File: dashboard/lib/routes/cbroker.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    return result;
+};
+
+/**
+ * @function postAllRegions
+ * Call to context broker and get all regions and status.
+ * @param {function} callback
+ */
+cbroker.postAllRegions = function (callback) {
```
Comment (pratid) [inline dashboard/lib/routes/cbroker.js:59]: To avoid confusion, I'd rename this to `retrieveAllRegions`

Comment (pratid) [inline dashboard/lib/routes/cbroker.js:59]: Again, this is more a function instead a method.

File: dashboard/lib/routes/cbroker.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+
+    var payload = {entities: [
+        {type: 'region', isPattern: 'true', id: '.*'}
+    ]};
```
Comment (pratid) [inline dashboard/lib/routes/cbroker.js:64]: Please narrow the attributes to be retrieved.

File: dashboard/lib/routes/cbroker.js [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+ * @param {Object} entities
+ * @return {Array}
+ */
+cbroker.parseRegions = function (entities) {
+
+    var result = [];
+
+    entities.contextResponses.forEach(function (entry) {
+        var type = entry.contextElement.type;
+        if (type === 'region') {
```
Comment (pratid) [inline dashboard/lib/routes/cbroker.js]: No more needed, as NGSI10 request specifies entities of type "region"

**Pull request closed without merging**